src/components/Demo/DemoCreate/DemoCreateReview/DemoPreview.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useCallback, useRef, FunctionComponent } from "react";
+import {
+  FrameConnector,
+  HostActions,
+  renderDocument,
+  FrameActions,
+} from "@govtechsg/decentralized-renderer-react-components";
+import { OpenAttestationDocument, utils } from "@govtechsg/open-attestation";
+
+type Dispatch = (action: HostActions) => void;
+
+interface DocumentPreview {
+  document: OpenAttestationDocument;
+}
+
+export const DocumentPreview: FunctionComponent<DocumentPreview> = ({ document }) => {
```

Comment:
think long ago had a discussion on whether to name interface exact same name as component, not sure about this wdyt

Reply:
can